subworkflows/local/phage.nf

```diff
@@ -0,0 +1,21 @@
+// INCLUDE MODULES
+include { VIRSORTER2 } from '../../modules/local/virsorter2/main.nf'  
```

Comment:
`include { VIRSORTER2 } from '../../modules/local/virsorter2/run/main.nf`

Reply:
resolved in [2fa40cd](https://github.com/cidgoh/nf-seqqc/pull/51/commits/2fa40cdfeba908da651b54951524de947c1c5dcb)